src/utils/validators.ts

```diff
@@ -1,17 +1,23 @@
 import {InvalidArgumentError} from '../errors/errors';
 
 export function validateCacheName(name: string) {
-  if (!name.trim()) {
+  if (isEmpty(name)) {
```

Comment:
I believe `isEmpty` returns `false` even if a passed value is an empty string, which we don't want?
i.e. `isEmpty("   ")` returns `false`.